docs/transaction-api.asciidoc

```diff
@@ -1,125 +1,56 @@
 [[transaction-api]]
-== Transaction API
+=== Transactions and Spans
 
 The following section contains information about:
 
-* <<transaction-endpoint>>
 * <<transaction-schema-definition>>
+* <<transaction-schema>>
+* <<transaction-span-schema>>
```

Comment:
I suggest to separate _transactions_ and _spans_ now that they are sent up independently and also described as separate event types. 

Also I'd reverse the order - first include the v2 specific schema, then the _common_  one.